2019-05-14-armagaddon/heka/generate_batches

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python
+# generate s3api delete object batches
+# https://docs.aws.amazon.com/cli/latest/reference/s3api/delete-objects.html
+import sys
+import json
+skeleton = '''{"Objects": [{"Key": "test1.txt"}],"Quiet": false}'''
+j = json.loads(skeleton)
```

Comment:
why is this pumped through the json decoder?
```suggestion
j = {"Objects": [{"Key": "test1.txt"}],"Quiet": False}
```

Reply:
I was just using https://docs.aws.amazon.com/cli/latest/reference/s3api/delete-objects.html#examples precisely but it doesn't really matter either way.